### ISSUE TITLE
spaces in abbrev

### ISSUE THREAD

**Issue by Alexey-T** [Alexey-T/Emmet-Pascal#14]
Notes on abbreviation formatting

When you get familiar with Emmet’s abbreviations syntax, you may want to use some formatting to make your abbreviations more readable. For example, use spaces between elements and operators, like this:

(header > ul.nav > li*5) + footer

But it won’t work, because space is a stop symbol where Emmet stops abbreviation parsing.

Many users mistakenly think that each abbreviation should be written in a new line, but they are wrong: you can type and expand abbreviation anywhere in the text

===

Lazarus - spaces handled. 
```
(header > ul.nav > li*5) + footer
```
->
```
<header>
	<ul class="nav">
		<li>|</li>
		<li>|</li>
		<li>|</li>
		<li>|</li>
		<li>|</li>
	</ul>
</header>
<footer>|</footer>
```

@rickard67

**Comment by Alexey-T** [comment]
This is why Emmet needs some indicators (like spaces) where it should stop parsing to not expand anything that you don’t need. If you’re still thinking that such formatting is required for complex abbreviations to make them more readable:

**Comment by rickard67** [comment]
Should have been fixed now in v1.06.

**Comment by Alexey-T** [comment]
Btw, you may find my code in EmmetHelper useful- it handles spaces ok
